src/main/java/edu/harvard/iq/dataverse/api/Datasets.java

```diff
@@ -1020,7 +1021,11 @@ public Response moveDataset(@PathParam("id") String id, @PathParam("targetDatave
                     ));
             return ok("Dataset moved successfully");
         } catch (WrappedResponse ex) {
-            return ex.getResponse();
+            if (ex.getCause() instanceof UnforcedCommandException) {
+                return ex.refineResponse("Use the query parameter forceMove=true to complete the move.");
```

Comment:
Can we add this response text to the Bundle>

Reply:
@sekmiller as I mentioned in person, I had this API only message in the bundle originally but decided to take it out because the rest of the API only messages are hard coded as English in the Java. Like we said, let's get a ruling at standup. 😄

Reply:
We decided that we want these API only messages in the bundle. We'll need a task in the future to make this change throughout the API, and in the meantime we'll pick these off for new APIs and for APIs that we're working on.

Reply:
In 926e955 I moved the English in the API I touched to the bundle. In 4c76181 I encouraged other developers to do the same with a note in the dev guide.